src/dispatch/dispatcher.rs

```diff
@@ -34,14 +31,12 @@ impl<'a, 'b> Dispatcher<'a, 'b> {
     ///
     /// Calling any method after `dispose` (including `dispose` itself) will
```

Comment:
This comment is no longer necessary. You can't call any method anyway because the object was moved :)